docs/source/kafka_check.rst

```diff
@@ -35,3 +35,37 @@ In case of min isr violations:
     CRITICAL: 1 partition(s) have the number of replicas in sync that is lower
     than the specified min ISR.
 
+Checking under replicated partitions
+====================================
+This kafka tool provides the ability to check number of under replicated partitions for all
+brokers in the cluster.
+
+The :code:`under_replicated` command checks if the number of under replicated partitions
+is equal to zero. This command will sum all numbers of under replicated
```

Comment:
Don't think 2nd line is necessary. Or may be, 'It will report the aggregated result of under replicated partitions of each broker if any.'